src/Stryker.Core/Stryker.Core/Reporters/Html/Realtime/Events/SseEvent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json;
+
+namespace Stryker.Core.Reporters.Html.Realtime.Events;
+
+public class SseEvent<T>
+{
+    public string EventName { get; init; }
+    public T Data { get; init; }
+
+    public override string ToString()
+    {
+        var data = JsonSerializer.Serialize(Data,
```

Comment:
It would be better to leave the transmission format up to the sender (either real time mutant handler or sseserver) and treat sseevent as a domain object